Sources/SwiftKotlinFramework/KotlinTokenizer.swift

```diff
@@ -619,28 +885,58 @@ public class KotlinTokenizer: SwiftTokenizer {
-                expression.newToken(.startOfScope, "(") +
-                exprs.map { tokenize($0) }.joined(token: expression.newToken(.delimiter, ", ")) +
-                expression.newToken(.endOfScope, ")")
+            // Trying to output example mutableListOf<String> if it's a declartion, or mutableListOf("xx", "yy") if it's an instance with values, not great
+            var hasType = false
+            for e in exprs {
+                switch e {
+                case let _ as IdentifierExpression:
+                    hasType = true
+                default:
+                    break
+                }
+            }
+            if hasType && exprs.count == 1 && !firstCharIsUpper(str:exprs[0].description) {
+                hasType = false
+            }
+            let middle = exprs.map { tokenize($0) }.joined(token: expression.newToken(.delimiter, ", "))
+            let mutable = [ expression.newToken(.identifier, "mutableListOf") ]
+            if hasType {
+                return
+                    mutable +
+                    [ expression.newToken(.startOfScope, "<") ] +
+                    middle +
+                    [ expression.newToken(.endOfScope, ">") ]
+            } else {
+                return
+                    mutable +
+                        [ expression.newToken(.startOfScope, "(") ] +
+                        middle +
+                        [ expression.newToken(.endOfScope, ")") ]
+            }
+            // have to output mutableMap since all swift arrays are mutable. Not great.
+//                expression.newToken(.identifier, "listOf") +
+//                expression.newToken(.startOfScope, "(") +
+//                exprs.map { tokenize($0) }.joined(token: expression.newToken(.delimiter, ", ")) +
+//                expression.newToken(.endOfScope, ")")
         case .dictionary(let entries):
             return
-                expression.newToken(.identifier, "mapOf") +
-                expression.newToken(.startOfScope, "(") +
+                expression.newToken(.identifier, "mutableMapOf") +
+                expression.newToken(.startOfScope, "<") +
                 entries.map { tokenize($0, node: expression) }
                     .joined(token: expression.newToken(.delimiter, ", ")) +
-                expression.newToken(.endOfScope, ")")
+                expression.newToken(.endOfScope, ">")
+//                expression.newToken(.startOfScope, "(") +
+//                expression.newToken(.endOfScope, ")")
         default:
             return super.tokenize(expression)
         }
     }
 
     open override func tokenize(_ entry: DictionaryEntry, node: ASTNode) -> [Token] {
         return tokenize(entry.key) +
-            entry.newToken(.space, " ", node) +
-            entry.newToken(.keyword, "to", node) +
-            entry.newToken(.space, " ", node) +
+//            entry.newToken(.space, " ", node) +
+            entry.newToken(.delimiter, ", ", node) +
```

Comment:
This was better before. The list of entries must be write as `"key" to "value"`

Reply:
I think that reading this change together with the above, you are using this piece of code to add the types. It may be a bug with the type but the purpose was to map this:

`let dict = ["value": 1]` to `val dict = mapOf("value" to 1)` but with the change would be `val dict = mutableMapOf<"value", 1>`